src/client/client.ts

```diff
@@ -311,6 +576,25 @@ export default class CTProtoClient<AuthRequestPayload, AuthResponsePayload, ApiR
     }
   }
 
+  /**
+   * Send all the enqueued buffer messages
+   */
+  private sendEnqueuedBufferMessages(): void {
+    while (this.enqueuedBufferMessages.length > 0) {
+      const enqueuedChunk= this.enqueuedBufferMessages.shift();
+
+      if ( !enqueuedChunk ) {
+        return;
+      }
+
+      const file = this.getUploadingFileById(enqueuedChunk.fileId);
+
+      if (file ) {
+        this.sendChunk( enqueuedChunk.message, enqueuedChunk.fileId );
```

Comment:
looks strange that the `sendChunk` does not accept the chunk or even chunk number